types/sap__approuter/xs-app.schema.d.ts

```diff
@@ -0,0 +1,100 @@
+// Original file: https://json.schemastore.org/xs-app.json
+// Online converter: https://bcherny.github.io/json-schema-to-typescript-browser/
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type SourceSchema =
```

Comment:
This package only exports types and no values; why did it need to be on DT?

Reply:
The type ComSapXsappSchema_82 is used in the [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc88925631fc646d7559fed8c19f58e39783f509/types/sap__approuter/index.d.ts#L106)

Reply:
Oops, sorry. GitHub hid the index from me so I thought this was the index.